plugin/pkg/shoot/validator/admission.go

```diff
@@ -843,6 +843,41 @@ func (c *validationContext) validateAdmissionPlugins(a admission.Attributes, sec
 	return allErrs
 }
 
+// For backwards-compatibility, we want to validate the scaleDownUnneededTime and scaleDownDelayAfterAdd only for newly created Shoot clusters.
+// Performing the validation for all Shoots would prevent already existing Shoots with the wrong spec to be updated/deleted.
+// Validation implies that ScaleDownDelayAfterAdd and ScaleDownUnneededTime shouldn't be less than a minute at the same time to avoid scaling cycles.
+func (c *validationContext) validateClusterAutoscaler(a admission.Attributes) field.ErrorList {
+	var allErrs field.ErrorList
+
+	if a.GetOperation() != admission.Create {
+		return nil
+	}
+
+	if c.shoot.Spec.Kubernetes.ClusterAutoscaler == nil || c.shoot.Spec.Kubernetes.ClusterAutoscaler.ScaleDownDelayAfterAdd == nil {
+		return nil
+	}
+
+	scaleDownDelayAfterAdd := c.shoot.Spec.Kubernetes.ClusterAutoscaler.ScaleDownDelayAfterAdd
+	scaleDownUnneededTime := c.shoot.Spec.Kubernetes.ClusterAutoscaler.ScaleDownUnneededTime
+	if scaleDownUnneededTime != nil && scaleDownDelayAfterAdd != nil {
```

Comment:
`scaleDownDelayAfterAdd != nil` can not happen here as there is early exit above